### PR TITLE
Enable fullscreen for videos

### PR DESCRIPTION
**Pull request by danigm**
Fix https://github.com/learningequality/kolibri-installer-android/issues/86

### CODE REVIEW

File: src/java/org/endlessos/FullScreen.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
+package org.endlessos;
```
Comment (rtibbles) [inline src/java/org/endlessos/FullScreen.java:1]:
```suggestion
package org.learningequality;
```

Could we rename the file as well?

File: src/main.py [hunk 2]
```diff
@@ -15,6 +15,10 @@
 
 PythonActivity = autoclass("org.kivy.android.PythonActivity")
 
+FullScreen = autoclass("org.endlessos.FullScreen")
```
Comment (rtibbles) [inline src/main.py:18]:
```suggestion
FullScreen = autoclass("org.learningequality.FullScreen")
```

File: src/java/org/endlessos/FullScreen.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        mChrome = new MyChrome(activity);
+    }
+
+    public static void configureWebview(PythonActivity activity) {
+        FullScreen fs = new FullScreen(activity);
+        fs.configure();
+    }
+
+    // Configure the WebView to allow fullscreen based on:
+    // https://stackoverflow.com/questions/15768837/playing-html5-video-on-fullscreen-in-android-webview/56186877#56186877
```
Comment (rtibbles) [inline src/java/org/endlessos/FullScreen.java:31]: This suggests that we need updates to the `AndroidManifest.xml` as well, or does the P4A generated manifest already include the appropriate keys?

**Review by rtibbles — COMMENTED** [review]
Not tested, but a naming tweak and a question.

File: src/java/org/endlessos/FullScreen.java (hunk 3 above)
Comment (danigm) [reply src/java/org/endlessos/FullScreen.java]: In my tests the fullscreen was working correctly with just this java class, I didn't need to do anything else to have the fullscreen working for videos.

**Review by rtibbles — APPROVED** [review]

**Pull request merged**